rollup.config.js

```diff
@@ -0,0 +1,19 @@
+import * as path from "path";
+import ts from "typescript";
+import rollupAlias from "rollup-plugin-alias";
+import typescript from "rollup-plugin-typescript";
+
+export default {
+	entry: "example/src/app.ts",
+	format: "iife",
+	dest: "example/bundle.js",
+	plugins: [
+		rollupAlias({
+			tslib: "node_modules/tslib/tslib.es6.js",
+			// angular: "node_modules/angular/angular.js"
+        }),
+		typescript({
+			typescript: ts
+		})
```

Comment:
you can check this one https://github.com/ezolenko/rollup-plugin-typescript2. rollup-plugin-typescript has old tslib.